client/src/Businesshours.jsx

```diff
@@ -3,24 +3,27 @@ import React from 'react';
 const businessHours = (props) => {
   return (
     <div>
-      <b>Hours</b>
-      <br></br>
-      <b>Mon</b> {props.businessHours.mon}
-      <br />
-      <b>Tue</b> {props.businessHours.tue}
-      <br />
-      <b>Wed</b> {props.businessHours.wed}
-      <br />
-      <b>Thu</b> {props.businessHours.thu}
-      <br />
-      <b>Fri</b> {props.businessHours.fri}
-      <br />
-      <b>Sat</b> {props.businessHours.sat}
-      <br />
-      <b>Sun</b> {props.businessHours.sun}
-      <br />
+      <span className="HoursHeading">
```

Comment:
Thanks Shane! Was thinking about that too but not sure how to render the Mon-Sun tag since my object doesn't contain it. Might look into doing another loop for that.

Reply:
on you app.jsx when you
if (key === 'mon' || key === 'tue' || key === 'wed' || key === 'thu' || key === 'fri' || key === 'sat' || key === 'sun') {
businessHours[key] = value;
}

i would do something like
businessHours[key] = {val: value, tag: key}
that way you have both the values and tag that you can just toUpperCase later